### PR TITLE
Function for importing large Database

### PR DESCRIPTION
**Pull request by adion-gorani**
Command line function to import large databases as phpMyAdmin is sometimes unreliable for large databases

### CODE REVIEW

File: bin/v-import-database [hunk 1]
```diff
@@ -31,24 +31,21 @@ source_conf "$HESTIA/conf/hestia.conf"
 #                    Verifications                         #
 #----------------------------------------------------------#
 
-check_args '2' "$#" 'DATABASE USER'
+check_args '3' "$#" 'DATABASE USER'
 is_format_valid 'database' 'user'
 is_system_enabled "$DB_SYSTEM" 'DB_SYSTEM'
 is_object_valid 'user' 'USER' "$user"
 is_object_unsuspended 'user' 'USER' "$user"
```
Comment (jaapmarcus) [inline bin/v-import-database:40]: Add 
is_object_valid 'database' 'DATABASE' "$database"

File: bin/v-import-database [hunk 2]
```diff
@@ -31,24 +31,21 @@ source_conf "$HESTIA/conf/hestia.conf"
...
 
 # Check owner existence
-owner=$(echo $database | cut -f 1 -d '_')
+owner=$user
```
Comment (jaapmarcus) [inline bin/v-import-database]: This whole check can be removed

**Review by jaapmarcus — APPROVED** [review]

**Comment by adion-gorani** [comment]
Modifications made

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
Made a few minor changes so the input validation works fine ..

**Pull request merged**